flink-architecture-tests/flink-architecture-tests-base/src/main/java/org/apache/flink/architecture/common/ImportOptions.java

```diff
@@ -18,6 +18,8 @@
 
 package org.apache.flink.architecture.common;
 
+import org.apache.flink.shaded.guava30.com.google.common.base.Predicate;
```

Comment:
why would you use guava for this when https://docs.oracle.com/javase/8/docs/api/java/util/function/Predicate.html exists?